docs/UserGuide.md

```diff
@@ -345,22 +321,120 @@ Examples:
 * `c-find John` returns `john` and `John Doe`.
 * `c-find alex david` returns `Alex Yeoh`, `David Li`.<br>
 
-#### 3.7 Locating persons based on matching tags: `c-tag-find`
+#### 3.5 Locating employees based on matching tags: `c-tag-find`
 
-Finds all active(unarchived) contacts that contain the KEYWORD(s) in their tags.
+Finds all employees whose tags contain the one or more `KEYWORD(s)`.
 
 Format: `c-tag-find KEYWORD [MORE_KEYWORDS] ...`
 
-* The search is case-insensitive. e.g `Friday` will match `friday`.
-* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
-* Only the tag names are searched.
-* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
-* Employees whose tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
- (i.e. `OR` search).  e.g. `Friday Monday` will return employees whose tags contain `Friday` or `Monday`.
+* The search is case-insensitive. e.g `Friday` will match `friday`.<br>
+* The order of the keywords does not matter. e.g. `monday friday` will match `monday friday`.<br>
+* Only the tag names are searched.<br>
+* Only full words will be matched. e.g. `Fri` will not match `Friday`.<br>
+* Employees whose tag(s) matching at least one keyword will be listed in the _Employee Directory_ pane
```

Comment:
Please change to Employee Directory section of Main View as we have discussed. Thanks

Reply:
It has been changed in the latest commit. Thanks!